### ISSUE TITLE
/draw Draw

### ISSUE THREAD

**Issue by enricoros** [enricoros/big-AGI#359]

**Comment by enricoros** [comment]
Folding the old Draw tickets into this one, so the ship-or-delete decision has the whole record in one place. All of them are now closed; what they carried is below.

- #289 Imagine overhaul: the removals and the one-shot prompt enhancer shipped. Unbuilt: a prompt matrix (2x2 desktop, 1x4 mobile, configurable 1-8) where each cell is a candidate prompt you can draw, optionally previewed in parallel, with hold-and-re-roll.
- #311 Drawing improvements: index ticket. Live items: exposing image generation to personas as a capability, and auto-draw after a reply, which is tracked separately in #669.
- #342 DALL-E prompts (@vadimkatsman): the verbatim-prompt switch and image refinement shipped. Unbuilt: generating several candidate prompts from one intent and picking or editing one before drawing.
- #355 DALL-E interface capabilities (@vadimkatsman): multi-image and refinement from a base image shipped. Unbuilt: a mask/inpainting UI (the server already accepts a mask, only the client and UI are missing) and a task-first surface - create/variate/edit/infill first, then the model - with a grid of prior generations to branch from. His projects/presets design is a separate subject and stays on that thread.
- #384 Story mode: Text -> Text[t] -> Image[t,n] plus a notion of Shape for the tensors, which is the Vector/Shaped generation requirement at the top of this ticket.
- #385 PromptFX: the prompt-matrix surface, concept screenshot on that ticket. src/apps/draw/promptfx/PromptFX.tsx has been an empty placeholder since 2024. Do not name it ImageFX, Google took that.
- #386 Ideas: the deck shipped (19 prompts, capybara included) and is unchanged since 2024. It needs a refresh before Draw ships again; the generator conversation is attached to #386 and linked from useDrawIdeas.tsx.

State of the app today: pages/draw.tsx and src/apps/draw/ exist and work, the nav entry is commented out, so Draw is reachable only by visiting /draw. Nothing above is scheduled. If Draw does not ship, all of it goes with it.

**Comment by enricoros** [comment]
Closing with a direction statement: the standalone Draw app is retired. Creation lives in the chat composer's draw mode, which covers generate-in-context, and the multi-generation surface will be Beam when it gets there. The ideas folded into this thread from the older tickets - prompt matrix, mask/inpainting UI, task-first surface, presets - stay on record here for that future.
